### DISCUSSION TITLE
Changing the audio pitch of the loaded swf

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2094] [General]
I had an application which load multiple external swf. 

Requirement : 
Can I change the pitch/tone of the loaded swf from the main.swf.
If yes, please guide.

**Comment by Adolio** [comment]
This library might help you: https://github.com/Adolio/Syrinx-Sound-Manager

**Reply by itlancer** [reply]
If you load external SWFs via `Loader` then you can get it "root" content (https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/Loader.html#content) as `Sprite`/`MovieClip` and try to change it `SoundTransform` https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/Sprite.html#soundTransform

**Reply by henke37** [reply]
The SoundTransform class doesn't do pitch, as cool as that would be.
